### PR TITLE
NDI-41: Style edit/view reports

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
# Description
This PR styles report and report template pages.

# Screenshots
## Report Templates
**Before**
image

**After**
image

## Report instance page
**Before**
image
**After**
image

## Report results
**Before**
before
**After**
after

# Technical Details
New Mixin is introduced for replacing button wrapper placeholder
```scss
@mixin civicrm-submit-buttons() {
  background: $crm-white !important;
  border-top: 1px solid $gray-light;
  margin: 0;
  padding: 20px;
}
```
# Comments
- BackstopJS tested
- Browserstack tested

### CODE REVIEW

File: scss/civicrm/contact/pages/_contact-report.scss [hunk 1]
```diff
@@ -5,6 +5,10 @@
     height: auto;
   }
 
+  .crm-submit-buttons {
+    @extend %submit-buttons-civi;
```
Comment (AkA84) [inline scss/civicrm/contact/pages/_contact-report.scss]: can we change this to be a mixin instead?

File: scss/civicrm/contact/pages/_reports.scss [hunk 2]
```diff
@@ -1,4 +1,49 @@
-#{civi-page('report-instance')} {
+body[class*='page-civicrm-report-'] {
```
Comment (AkA84) [inline scss/civicrm/contact/pages/_reports.scss]: what's wrong with `civi-page`?

File: scss/civicrm/contact/pages/_reports.scss [hunk 3]
```diff
@@ -1,4 +1,49 @@
...
+
+  #crm-container {
```
Comment (AkA84) [inline scss/civicrm/contact/pages/_reports.scss]: why is this necessary?

File: scss/civicrm/contact/pages/_reports.scss (hunk 2 above)
Comment (mukeshcompucorp) [reply scss/civicrm/contact/pages/_reports.scss]: Custom report pages are not having class `page-civicrm-report-instance`, and `body[class*='page-civicrm-report-']` rule applies to all report instance pages, weather it's default report instance or a custom report instance. So, I used this instead to make sure all page gets the style properly

File: scss/civicrm/contact/pages/_reports.scss (hunk 3 above)
Comment (mukeshcompucorp) [reply scss/civicrm/contact/pages/_reports.scss]: Core CiviCRM uses `#crm-container` to style the tables so, instead of adding `!important`s in different css, I chose to take same container as core civicrm to override the style.

File: scss/civicrm/contact/pages/_contact-report.scss (hunk 1 above)
Comment (mukeshcompucorp) [reply scss/civicrm/contact/pages/_contact-report.scss]: Ok, I will convert this one.
